jupyter_scheduler/models.py

```diff
@@ -25,6 +25,7 @@ class RuntimeEnvironment(BaseModel):
     metadata: Optional[Dict[str, str]]  # Optional metadata
     compute_types: Optional[List[str]]
     default_compute_type: Optional[str]  # Should be a member of the compute_types list
+    utc_only: Optional[bool]
```

Comment:
The naming here seems a bit odd to me. Should we rename to `timezones` or something related and have it True for OSS?

Reply:
I thought about using `timezones`, but I rejected it, thinking that the user would expect `timezones` to be set to a list of time zones. We could say `supports_timezones` if we want this to be a positive, although we'd have to interpret "unset" as "true" to avoid breaking existing envs.

Reply:
I support preserving backwards compatibility by default, especially now that we're on 1.0.